doc/en/user/source/styling/sld/extensions/pointsymbols.rst

```diff
@@ -286,7 +286,75 @@ For example:
       </Graphic>
     </PointSymbolizer>
 
-In this example an SVG graphic is being used, so the size is specified explicitly. 
+In this example an SVG graphic is being used, so the size is specified explicitly.
+
+Parameterized (like QGIS) SVG images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Just like QGIS, Geoserver can handle SVG images in which parts of the SVG-attributes are 
+named parameters as defined in the `SVG Parameters 1.0 spec <https://www.w3.org/TR/SVGParamPrimer/>`_.
+
+A QGIS SVG like: `poi_peak.svg <https://github.com/qgis/QGIS/blob/master/images/svg/symbol/poi_peak.svg>`_ has as SVG source:
+
+.. code-block:: xml
+
+   <svg enable-background="new 0 0 580 580" height="580" viewBox="0 0 580 580" width="580" xmlns="http://www.w3.org/2000/svg">
+   <path d="m290.565 67.281l-255.498 442.534-1.087 1.885 511.229.393 2.18.002z" fill="param(fill)" 
+    fill-opacity="param(fill-opacity)" stroke="param(outline)" stroke-opacity="param(outline-opacity)" stroke-width="param(outline-width)"/>
+   </svg>
+
+The 'param'-constructs mean that you can define the parameters: fill, fill-opacity, outline, outline-opacity and outline-width as so called SVG URL references,
+e.g. a refence to this image with red fill would be: ``poi_peak.svg?fill=#FF0000``.
+
+Note: when you edit svg's yourself (e.g. in Inkscape) you should save your svg's as 'simple svg'.
```

Comment:
We try to use ` characters (monospace) for named things as they are displayed different, and not to quote things like "simple svg".

Reply:
Our [writing guidelines](https://docs.geoserver.org/latest/en/docguide/style.html) try and avoid the use of "we" and "you" and instead provide direct commands.

```
Note: When editing SVG files (e.g. in Inkscape) save using the `simple svg` format.
``